src/taco-utils/package.json

```diff
@@ -35,7 +35,9 @@
         "semver": "2.3.1",
         "unorm": "1.3.3",
         "windows-no-runnable": "0.0.6",
-        "wrench": "^1.5.8"
+        "wrench": "^1.5.8",
+        "lodash": "^3.10.1",
```

Comment:
It looks like lodash is used by telemetry, telemetryHelper, and promisesUtils. Wrench is used by projectHelper.